eks/cuda-vector-add/cuda-vector-add.go

```diff
@@ -0,0 +1,254 @@
+package cuda_vector_add
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+
+	eks_tester "github.com/aws/aws-k8s-tester/eks/tester"
+	"github.com/aws/aws-k8s-tester/eksconfig"
+	k8s_client "github.com/aws/aws-k8s-tester/pkg/k8s-client"
+	"github.com/aws/aws-k8s-tester/pkg/timeutil"
+	"go.uber.org/zap"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/exec"
+)
+
+// Config defines Cuda-Vector-Add configuration.
+type Config struct {
+	Logger *zap.Logger
+	Stopc  chan struct{}
+
+	EKSConfig *eksconfig.Config
+	K8SClient k8s_client.EKS
+}
+
+const (
+	cuda_vector_add_pod_name = "vector-add"
+)
+
+// Tester defines Cuda-Vector-Add tester.
+type Tester interface {
+	// Create method deploy Cuda-Vector-Add.
+	Create() error
+	// Delete method delete Cuda-Vector-Add.
+	Delete() error
+}
+
+func New(cfg Config) eks_tester.Tester {
+	cfg.Logger.Info("creating tester", zap.String("tester", reflect.TypeOf(tester{}).PkgPath()))
+	return &tester{cfg: cfg}
+}
+
+type tester struct {
+	cfg Config
+}
+
+func (ts *tester) AggregateResults() error {
```

Comment:
Can we move `Aggregate...` to the bottom?

Reply:
Fixed.